snapcraft.yaml

```diff
@@ -17,4 +17,16 @@ parts:
       - grub-pc-bin
       - grub-common
     plugin: make
+  uuidgen:
+    plugin: nil
+    stage-packages: [ uuid-runtime ]
+    stage: [ usr/bin/uuidgen, usr/share/doc/uuid-runtime/copyright ]
 
+hooks:
+  prepare-device:
+    environment:
+    # TODO: define your model's API key here
```

Comment:
We should not have a `TODO` as it applies only if you are forking the gadget. Something like "If you are forking the gadget you can define...".